css/style.css

```diff
@@ -60,4 +60,92 @@
 .button--danger {
 	color: #ffffff;
 	background-color: #ce4141;
-}
\ No newline at end of file
+}
+/*Comodities */
+html {
+  font-family: "Roboto", sans-serif;
+  margin: 20px;
+}
+@media (max-width: 768px) {
+  img {
+    width: 100%;
+    height: 100%;
+    max-height: 233px;
+    max-width: 367px;
+  }
+  .carrouselContainer {
+    width: 95%;
+    max-width: 1000px;
+    margin: 0 auto;
+    box-sizing: border-box;
+  }
+  .title-gallery {
+    width: 25px bold;
+  }
+  .line {
+    width: 90px;
+    height: 1px;
+    margin-left: 0;
+    background-color: #63ccff;
+    border: 0;
+  }
+  .gallery {
+    padding-top: 10%;
+    width: 100%;
+    display: flex;
+    box-sizing: border-box;
+  }
+  .gallery__bottom {
+    font-size: 50px;
+    width: 16%;
+    height: 24px;
+    font-weight: bold;
+    position: relative;
+  }
+  .gallery__bottom:hover {
+    cursor: pointer;
+    transform: scale(1.2, 1.2);
+  }
+  .back {
+    margin-top: 74.1px;
+    left: 17%;
+    color: #fefefe;
+    position: relative;
+  }
+  .ahead {
+    margin-top: 74.1px;
+    right: 41px;
+    color: #fefefe;
+  }
+  .pointer {
+    border-radius: 50%;
+    display: flex;
+    padding-left: 42%;
+    margin: 10px;
+    list-style-position: initial;
+    list-style-type: disc;
+  }
+  .pointer__point {
+    border-radius: 50%;
+    padding-left: 0%;
+    margin-left: 6%;
+  }
+  .pointer:hover {
+    cursor: pointer;
+    transform: scale(1.2, 1.2);
+  }
+  .button-gallery {
```

Comment:
Have you thought about using the Button component created by Christian?

Reply:
+ 1, please use the preexisting button component